tamr_client/attribute/__init__.py

```diff
@@ -0,0 +1,14 @@
+from . import sub
```

Comment:
It looks like you are using relative imports in the `__init__.py` files.  I think we determined that relative imports are generally okay, but up until now were not being used in the `TC` codebase.  Is there a good reason to use 
`from . import sub` vs `from tamr_client.attribute import sub`?

If that is the case, I think it is worth adding a line to the Internal Imports section of the Style Guide that clarifies when relative imports are appropriate (only in `__init__.py` if I understand this PR).

Reply:
We could just delegate to [PEP 8's explanation of imports](https://www.python.org/dev/peps/pep-0008/#imports) i the Style Guide, specifically:

> explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose

Reply:
I think this is an example where `from . import sub` and `from tamr_client.attribute import sub` are both reasonable choices.  The former is not too difficult to understand and the latter isn't so verbose as to be problematic.  Personally I prefer the absolute import but I'm fine with either.

Reply:
Changed imports to all be absolute.